src/stories/Library/nav-teaser/NavTeaser.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from "react";
+
+type NavTeaserProps = {
+  title: string;
+  subtitle?: string;
+};
+
+const NavTeaser: FC<NavTeaserProps> = ({ title, subtitle }) => {
+  return (
+    <article className="nav-teaser">
+      <a href="#">
+        <h1 className="nav-teaser__title">{title}</h1>
+
+        {subtitle ? <p className="nav-teaser__subtitle">{subtitle}</p> : ""}
+
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="33"
+          height="8"
+          viewBox="0 0 33 8"
+          fill="black"
+        >
+          <path d="M32.287 4.35356C32.4823 4.15829 32.4823 3.84171 32.287 3.64645L29.105 0.464469C28.9098 0.269206 28.5932 0.269206 28.3979 0.464469C28.2027 0.659731 28.2027 0.976313 28.3979 1.17158L31.2264 4L28.3979 6.82843C28.2027 7.02369 28.2027 7.34027 28.3979 7.53554C28.5932 7.7308 28.9098 7.7308 29.105 7.53554L32.287 4.35356ZM-4.37114e-08 4.5L31.9335 4.5L31.9335 3.5L4.37114e-08 3.5L-4.37114e-08 4.5Z" />
+        </svg>
```

Comment:
I think we should use `  <ArrowSmallRight />` or similar to avoid these large svg pastes.

Reply:
Good point - I've added it now. It didnt exist already.